hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/StreamSync.java

```diff
@@ -402,7 +402,9 @@ public Pair<Option<String>, JavaRDD<WriteStatus>> syncOnce() throws IOException
         .build();
     String instantTime = metaClient.createNewInstantTime();
 
+    long startInput = System.currentTimeMillis();
     InputBatch inputBatch = readFromSource(instantTime, metaClient);
+    LOG.error("Time to read from source : " + (System.currentTimeMillis() - startInput));
```

Comment:
Use `HoodieTimer` to track execution time?

Reply:
This was from my own debugging of slow tests. I will revert